assets/app/view/game/split.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+
+module View
+  module Game
+    class Split < Snabberb::Component
```

Comment:
what does this look like? can you just model this with a choose action?

Reply:
This is the button for split. It's a action you can do on a corp during the stock round.